chipsec/modules/common/rtclock.py

```diff
@@ -121,7 +121,7 @@ def check_rtclock(self):
 
         return self.rc_res.getReturnCode(res)
 
-    def run(self, module_argv):
+    def run(self, *module_argv: str) -> int:
```

Comment:
This I think is causing tests to fail. All of these should be:
```suggestion
    def run(self, module_argv: List[str]) -> int:
```
I think